pom.xml

```diff
@@ -519,6 +519,7 @@
                                 @{argLine}
                                 --add-opens java.base/java.util=ALL-UNNAMED
                                 --add-opens java.base/java.lang=ALL-UNNAMED
+                                --add-exports=java.xml.crypto/com.sun.org.slf4j.internal=ALL-UNNAMED
```

Comment:
Not sure what to make of this?

Reply:
yes, I had a wrong import, I noticed the failed build and [fixed]([b1809cc](https://github.com/aws-powertools/powertools-lambda-java/pull/1306/commits/b1809ccbd7d9dea551fcbbc9226abe85a85ba585))